tests/unit/amo/reducers/test_collections.js

```diff
@@ -829,61 +828,6 @@ describe(__filename, () => {
     });
   });
 
-  describe('updateCollection', () => {
```

Comment:
I removed this because there isn't anything left to test here, but I noticed that we don't have coverage for testing just action creators which don't result in any state changes. I suppose there's not much to test, other than that the return result is what we expect, so maybe this is fine. But it seemed weird to add code and not also add a test for it.

Reply:
> I removed this ... but I noticed that we don't have coverage for testing just action creators which don't result in any state changes.

We do. Any action creator that doesn't result in state changes exists solely to trigger a saga, which would be covered in [a test like this](https://github.com/mozilla/addons-frontend/blob/3d510e53b1011d83b3665a3de2aaecffaedc5da7/tests/unit/amo/sagas/test_collections.js#L332).